content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -237,8 +237,13 @@ uses packet processing logic (such as Linux iptables) to define _virtual_ IP
 addresses which are transparently redirected as needed.
 
 When clients connect to the VIP, their traffic is automatically transported to an
-appropriate endpoint. The environment variables and DNS for Services are actually
-populated in terms of the Service's virtual IP address (and port).
+appropriate endpoint. Actually, the kubelet on each node populates environment variables
+for containers based on the port number for the Service and on the virtual IP address
+(_cluster IP_), if any, that has been assigned.
```

Comment:
Grammatical nit. We are talking about the assignment of port number & virtual IP address = 2 things. The appropriate verb would be "have"
```suggestion
(_cluster IP_), if any, that have been assigned.
```